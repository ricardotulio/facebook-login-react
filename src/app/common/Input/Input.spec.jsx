import React from 'react'
import renderer from 'react-test-renderer'

import Input from './Input'

test('Properties are passed to input element', () => {
  const input = renderer.create(<Input id="input-id" name="input-name" />)
  let tree = input.toJSON()

  expect(tree).toMatchSnapshot()
})