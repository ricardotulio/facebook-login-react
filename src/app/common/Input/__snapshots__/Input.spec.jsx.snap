// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties are passed to input element 1`] = `
<input
  id="input-id"
  name="input-name"
/>
`;
